content/post/career-ladders-part-1.md

```diff
@@ -0,0 +1,39 @@
+## What Is a Career Ladder, and Why Do I Need One?
+A career ladder is a framework that documents the path(s) an individual can take as they develop within their company. It describes different roles (hopefully both individual contributor and manager roles), outlines the expectations for each, and provides sufficiently detailed information to map a path from one role to another. A career ladder can include hard skills, as well as an outline of *how* one should work in each role and the type of impact that role should have.
+
+When I started my career, I didn’t have a career ladder, and didn't know what I was missing. I naively assumed that if I did good work, I would be acknowledged for it. It mostly worked - I got promoted a few times, shifted to an entirely new field, and got promoted a few more times. Most of those promotions came as a pleasant surprise - while I love learning new things and generally tried to work on whatever was most useful to the company, I never had much guidance on what the “next level” looked like.
+
+I didn’t appreciate the value of a career ladder until I started to lead a team. Having helped each person on my team identify development opportunities and work toward promotions, I now view a career ladder as critical to managing a team in a productive, fair way.
+
+### Keep Your Team Happy and Motivated
+Analysts, of all people, want to know how they are being measured and how to reach the next level. Clearly outlining the path ahead is critical to keeping strong performers engaged and growing. Sometimes they explicitly want to know how to get the next promotion, but they also want tangible ways increase their impact, measure progress, and think about future options. Recognition matters, and a career ladder enables recognition of both big steps (promotions) and smaller steps (acknowledging that someone has demonstrated proficiency with one skill even if they have further to go toward the next level). It also provides a context for feedback that is productive, not personal.
+
+### Make the Implicit Explicit
+Defining a career ladder is also an opportunity for to define what types of work are valuable on the team. There are hard skills you expect to see improve over time, but much more goes into being a successful contributor to the team than having deep knowledge of tools and techniques. Rather than counting on your team to understand what maturity in a role looks like, a good career ladder spells it out in tangible terms. This is especially important when your team members are diverse in terms of race, gender, age, cultural background, or other identities, or when they just have different experiences. So, basically, it's always important. Explicitly articulating expectations of how one should work at each level is also an opportunity to turn [glue work](https://www.locallyoptimistic.com/post/glue-work/) into promotable work by clarifying what collaboration, team process work, and mentorship look like at various levels.
```

Comment:
One thing I've struggled with on this point is how concrete to make the expectations. I've had team members ask for metrics like "x pull requests per week" and other "truly objective" metrics for showing development. My issue has been that for knowledge workers, there are no "truly objective" measures of when someone is doing a good job. So while it's good to have these things written down, there's no escaping some measure of judgement.

IDK ... what do you think? Worth addressing?